src/site/layouts/page.drupal.liquid

```diff
@@ -73,6 +73,8 @@
         </div>
       </div>
     </main>
+    <!-- Up To Top Button -->
+  {% include "src/site/components/up_to_top_button.html" %}
```

Comment:
We might want to indent this line to the right a bit. Could you try this locally if you haven't already and confirm it aligns correctly for various pages? 🙂

Reply:
done!